plugins/codecs/ndlz/test_ndlz.c

```diff
@@ -0,0 +1,264 @@
+/*
+    Copyright (C) 2014  Francesc Alted
```

Comment:
Not sure if you want to use `The Blosc2 Developers` here as copyright which I have seen in other places.